shared/routes/Repl/Viewer.html

```diff
@@ -17,10 +19,27 @@
 </div>
 
 <style>
+	.iframe-container {
+		border-top: 1px solid #ccc;
+	}
+
 	iframe {
 		width: 100%;
-		height: 100%;
+		height: calc(100vh - 3em);
 		border: none;
+		background-color: white;
```

Comment:
You should set `color` as well. Otherwise people using Firefox with a dark themed window decoration get a bright gray on white background...